packages/react-components/src/components/styles/VideoTile.styles.ts

```diff
@@ -12,6 +12,10 @@ export const rootStyles: IStyle = {
   borderRadius: theme.effects.roundedCorner4
 };
 
+export const isSpeakingStyles: IStyle = {
+  border: '4px solid #0078D4'
```

Comment:
The color for this border should also be picked from the theme palette. This hard-coded color may not look good on inverted backgrounds etc.

Reply:
Yes. I dreamt of this last night :D